injector/LogInjector.py

```
@@ -103,6 +104,10 @@ def visit_AnnAssign(self, node):
         else:
             return self.processNode(node).injectLogsTypeB()
 
+    def visit_Expr(self, node):
```

Ruff (0.8.2)

Redefinition of unused `visit_Expr` from line 95

(F811)